### PR TITLE
README work

### PR DESCRIPTION
**Pull request by mmacy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
-# Helm Charts for Kubernetes Service Catalog on Azure
+# Helm charts for Kubernetes Service Catalog on Azure
```
Comment (jeremyrickard) [inline README.md]: Could you please revert back to title case for the section headers?

File: README.md [hunk 2]
```diff
@@ -50,39 +48,37 @@ After you've added the Azure charts repository, you'll have access to the charts
...
 local/wordpress 	0.6.12 	Web publishing platform for building blogs and ...
 stable/wordpress	0.6.10 	Web publishing platform for building blogs and ...
 ```
 
-Notice that `azure/wordpress` is listed above. This is the chart that you see in this
-Github repository.
+The `azure/wordpress` chart shown in the output above is the chart that you see in this Github repository.
 
-# Creating a New Chart
+## Creating a new chart
```
Comment (jeremyrickard) [inline README.md]: Title Case

File: README.md [hunk 3]
```diff
@@ -20,25 +18,25 @@ in tandem.
...
-and [Open Service Broker for Azure](https://github.com/Azure/open-service-broker-azure) 
+and [Open Service Broker for Azure](https://github.com/Azure/open-service-broker-azure)
 installed on it.
 
-Please see [the prerequisities guide](./docs/prerequisities/README.md) for 
+Please see the [prerequisities guide](./docs/prerequisities/README.md) for
 details on how to install all of these prerequisities.
 
-# Installing Charts
+## Installing charts
```
Comment (jeremyrickard) [inline README.md]: Title Case

**Comment by mmacy** [comment]
Should be all fixed up, @jeremyrickard, thanks for the review.

**Comment by jeremyrickard** [comment]
Docs only, skipping CI.

**Pull request merged**